src/deluge/gui/menu_item/patch_cable_strength.h

```diff
@@ -31,7 +31,7 @@ class PatchCableStrength : public Decimal, public MenuItemWithCCLearning {
 	[[nodiscard]] int32_t getMaxValue() const final { return kMaxMenuPatchCableValue; }
 	[[nodiscard]] int32_t getNumDecimalPlaces() const final { return 2; }
 	virtual int32_t getDefaultEditPos() { return 2; }
-	MenuPermission checkPermissionToBeginSession(Sound* sound, int32_t whichThing, MultiRange** currentRange) override;
+	MenuPermission checkPermissionToBeginSession(Sound* sound, int32_t whichThing, MultiRange** currentRange);
 	virtual ParamDescriptor getDestinationDescriptor() = 0;
```

Comment:
not sure why but it wouldn't compile unless I removed override from this function...